src/node/node_info_network.h

```diff
@@ -9,6 +9,15 @@
 
 namespace ccf
 {
+  enum class EndorsementType
+  {
+    Node,
```

Comment:
I initially named those "Self" and "Service" but then renamed them to match the existing terminology.